dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.springBoot40ServerSupport.servlet61-1.0.feature

```diff
@@ -1,12 +1,12 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
-symbolicName=io.openliberty.springBoot40ServerSupport.servlet60-1.0
+symbolicName=io.openliberty.springBoot40ServerSupport.servlet61-1.0
```

Comment:
Previously you updated the repository.xml files in com.ibm.ws.kernel.feature.resolver_fat as well when you changed / added this feature.